src/index.html

```diff
@@ -9,6 +9,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="images/logo.png"
+        alt="MOYO_logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="list_item">
+            <a href="#" class="list_link is-active">Apple</a>
```

Comment:
add more meaningful links
```suggestion
            <a href="#apple" class="list_link is-active">Apple</a>
```